package.json

```diff
@@ -110,6 +110,7 @@
     "tz-lookup": "^6.1.25",
     "urql": "^4.1.0",
     "use-resize-observer": "^9.0.2",
+    "usehooks-ts": "^3.1.1",
```

Comment:
`usehooks-ts` is a handy dandy utility library that we've been using in other projects, here it's brought in for it's React local storage interface but it has many other utilities we can make use of